zello-sdk-sample/src/main/java/com/zello/sdk/sample/ListAdapter.java

```diff
@@ -181,6 +181,7 @@ private static String statusToText(Context context, ContactStatus status) {
 		return context.getResources().getString(id);
 	}
 
+	@SuppressWarnings("SwitchStatementWithTooFewBranches")
```

Comment:
Since this is mostly a stylistic warning, I would think we could turn it off globally if we don't care about it. Or are there instances where we would want to see this warning?

Reply:
Not the most useful warning out there for sure, so I agree it's hard to think of a scenario where it helps at all. I would just follow the suggestion and use an `if` instead.